### PR TITLE
feat(add): Shyugj DoorSensor-ZB3.0

### PR DESCRIPTION
**Pull request by bobrippling**
This Pull Request adds support for the Shyugj Door Sensor. There is currently no support for this vendor, so a definition file has been created for it.

### CODE REVIEW

**Comment by Koenkk** [comment]
Could you provide the data/database.db entry of this device?

File: src/devices/shyugj.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import {battery, iasZoneAlarm} from '../lib/modernExtend';
+import {DefinitionWithExtend} from '../lib/types';
+
+const definitions: DefinitionWithExtend[] = [
+    {
+        zigbeeModel: ['DoorSensor-ZB3.0'],
+        model: 'DoorSensor-ZB3.0',
```
Comment (Koenkk) [inline src/devices/shyugj.ts:7]: Is there a better model number written on the device or product page?

Comment (bobrippling) [reply src/devices/shyugj.ts:7]: The only other identifier I have is an item number on the setup documentation, `S901D-ZG`, if that feels more useful?

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**